src/main/scala/com/lightbend/kafka/scala/streams/StreamsBuilderS.scala

```diff
@@ -17,62 +17,65 @@ import scala.collection.JavaConverters._
 
 /**
  * Wraps the Java class StreamsBuilder and delegates method calls to the underlying Java object.
- */ 
-class StreamsBuilderS {
-
-  val inner = new StreamsBuilder
+ */
+class StreamsBuilderS(streamsBuilder:StreamsBuilder) {
```

Comment:
Can u please keep the name `inner` here ? We use this name consistently in other classes as well.